instance/models/mixins/storage.py

```diff
@@ -313,8 +313,9 @@ def _create_bucket(self, max_tries=4, retry_delay=15, location=None):
                 )
                 break
             except ClientError as e:
-                if e.response.get('Error', {}).get('Code') == 'EntityAlreadyExists':
+                if e.response.get('Error', {}).get('Code') == 'BucketAlreadyOwnedByYou':
```

Comment:
@viadanna, will there be any scenarios where handling the `EntityAlreadyExists` exception will be needed?

Reply:
Not in this context, as after the refactor it just tries to create a bucket. In `us-east-1` it's simply ignored and on other zones it returns the value tested when the bucket already exists.